free-2.md

```diff
@@ -58,12 +95,22 @@ making language design decisions), as opposed to "systems" aspects of the
 project (e.g., implementing a complicated semantics that doesn't require a lot
 of language design)?_
 
+I think it would be a 60-40 split in favor of language design.  With a very 
+rigid syntax and structure, it would take a lot of work to get that part down
+exactly.  I think the NLP and parsing would be very non-trivial, but the language
+design would trump that.
```

Comment:
And you could definitely impact that ratio by deciding how rigid to make the syntax and structure - a more rigid structure will require less NLP.